tests/_support/Page/AbstractPage.php

```diff
@@ -18,16 +18,41 @@ class AbstractPage
 	 */
 	public static $h1 = ['css' => 'h1'];
 
+	/**
+	 * @var string
+	 */
+	public static $clientId = '#client_id';
+
 	/**
 	 * @var string
 	 */
 	public static $buttonSave = 'Save';
 
+	/**
+	 * @var string
+	 */
+	public static $buttonSaveAndClose = 'Save & Close';
+
 	/**
 	 * @var array
 	 */
 	public static $messageContainer = '#system-message-container';
 
+	/**
+	 * @var array
+	 */
+	public static $sysMessageContainer = ['id' => 'system-message-container'];
```

Comment:
@NguyenTrang091296  can you please help Oscar change the code follow 2.4.0 codeception style. thanks you

Reply:
Hi @OscarIboy. You can change the code follow 2.4.0 codeception style. I have writen  it on file https://redweb.atlassian.net/wiki/spaces/REDSHOP/pages/675709139/redSHOP+Automation